File: 백준/Bronze/28417. 스케이트보드/스케이트보드.py
n = int(input())

ans = 0
for _ in range(n):
    scores = list(map(int, input().split()))
    one = max(scores[0], scores[1])
    two, three = sorted(scores[2:])[-1], sorted(scores[2:])[-2]
    ans = max(ans, one + two + three)

print(ans)

File: 백준/Bronze/28417. 스케이트보드/스케이트보드.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        int ans = 0;

        for (int i = 0; i < n; i++) {
            int[] scores = new int[7];
            for (int j = 0; j < 7; j++) {
                scores[j] = sc.nextInt();
            }
        

            int one = Math.max(scores[0], scores[1]);
            Arrays.sort(scores, 2, 7);
            int two = scores[6];
            int three = scores[5];

            ans = Math.max(ans, one + two + three);
        }

        System.out.println(ans);
    }
}